src/components/ProcedureTask.js

```diff
@@ -17,6 +17,37 @@ const styles = theme => ({
 const ProcedureTask = props => {
     const { classes } = props;
 
+    function getProcedureContent() {
+        return (
+            <Grid 
+             item 
+             xs={3} 
+             className={classes.grid} 
+             style={{justifyContent: 'space-around'}}>
+                <TransitionsModal data={'Schedule'} />
+                <TransitionsModal data={'Edit'} />
+                <TransitionsModal data={'More'} />
+                <PlayCircleFilledWhiteRoundedIcon />
+            </Grid>
+        )
+    }
+
+    function getTaskContent() {
+        return (
+            <Grid 
+             item 
+             xs={3} 
+             className={classes.grid} 
+             style={{justifyContent: 'space-around'}}>
```

Comment:
Maybe it made more sense to separate 'ProcedureTask', 'getProcedureContent' and 'getTaskContent' on three components.